doc/mesh-adapter-api.txt

```diff
@@ -0,0 +1,137 @@
+BlueZ D-Bus Mesh Adapter API description
+********************************
+
+Mesh Network Hierarchy
+======================
+Service		org.bluez.mesh
+Interface	org.bluez.mesh.AdapterManager1
+Object path	/org/bluez/mesh/adapter
+
+Methods:
+	uint64 token Add(string io_type, dict config)
+
+	    This is method that an application has to call to
+	    add a new mesh adapter with an IO type.
+
+	    The config parameter is a dictionary with configuration
+	    options specific for the chosen IO type. In case of "tcpserver",
```

Comment:
I think every IO type should have a defined tuple of settings that are meant to be unique. For example, in case of UART adapter this would be just the device name, while for TCP server this would be (UUID, DeviceKey, NetworkKey).

Let's document this for each io type separately.